install-panel.sh

```diff
@@ -744,118 +756,59 @@ function configure_nginx {
       ln -s /etc/nginx/sites-available/pterodactyl.conf /etc/nginx/sites-enabled/pterodactyl.conf
   fi
 
-  # restart nginx
-  if [ "$CONFIGURE_LETSENCRYPT" == true ] || { [ "$CONFIGURE_LETSENCRYPT" == false ] && [ "$ASSUME_SSL" == false ]; }; then
+  if "$ASSUME_SSL" && "$CONFIGURE_LETSENCRYPT"; then
```

Comment:
Same here about the square brackets. Also, doesn't this only restart if `ASSUME_SSL` and `CONFIGURE_LETSENCRYPT` are both true? For which cases do we need to restart Nginx? It should only be when `ASSUME_SSL` and `CONFIGURE_LETSENCRYPT` are both false (user is not using HTTPS) since you said certbot restarts Nginx automatically and when user is not using Let's Encrypt but they are planning on using SSL, they need to modify their config file before restarting Nginx manually. Am I missing something obvious here?

Reply:
They can't be both true.

Reply:
Which variables cannot be true simultaneously?

Reply:
`ASSUME_SSL` and `CONFIGURE_LETSENCRYPT` can't be true at the same time.

Reply:
Can they not? I think in the `ask_letsencrypt` function they are both set to true. See https://github.com/Linux123123/pterodactyl-installer/blob/95a1f89bf90347fc936d1375bde7be4d32d49d41/install-panel.sh#L196

Reply:
Yes, you are right. It's an issue. It was left behind, because I changed the way how assuming ssl works. There it should only be set `CONFIGURE_LETSENCRYPT`.

Reply:
So what purpose do the variables have now? `ASSUME_SSL` should only be true if `CONFIGURE_LETSENCRYPT` is false and the user then wants to use their own certificate. If `CONFIGURE_LETSENCRYPT` is true the user wants to use Let's Encrypt automatically but the `ASSUME_SSL` should **not** be true. And if both are false, just pure HTTP. Am I correct?

Reply:
Yes, you are correct.

Reply:
Alright, considered resolved then.

Reply:
Nevermind, still about the brackets.

Reply:
Fixed.